### PR TITLE
chore: Improving error message

### PR DESCRIPTION
**Pull request by bahaa-ghazal**
Improving error message when calling mender-artifact on a device without mender

Ticket: MEN-7832

### CODE REVIEW

**Comment by lluiscampos** [comment]
HI @bahaa-ghazal and congratulations with your first contribution! Please use our company e-mail in your git author and signature: `Signed-off-by: Bahaa Ghazal <bahaaghazal2003@gmail.com>`

File: cli/write.go [hunk 1]
```diff
@@ -880,10 +880,12 @@ func getDeviceSnapshot(c *cli.Context) (string, error) {
...
-		`'mender_snapshot="mender snapshot"`+
-			`; which mender-snapshot 1> /dev/null && mender_snapshot="mender-snapshot"`+
-			`; [ $(id -u) -eq 0 ] || sudo_cmd="sudo -S"`+
-			`; $sudo_cmd /bin/sh -c "echo `+sshInitMagic+`; $mender_snapshot dump" | cat'`,
+		`'[ $(id -u) -eq 0 ] || sudo_cmd="sudo -S"`+
+			`; if which mender-snapshot 1> /dev/null`+
+			`; then $sudo_cmd /bin/sh -c "echo `+sshInitMagic+`; mender-snapshot dump" | cat`+
+			`; elif which mender 1> /dev/null`+
+			`; then $sudo_cmd /bin/sh -c "echo `+sshInitMagic+`; mender snapshot dump" | cat`+
+			`; else echo "Mender not found: Please check that Mender is installed" >&2 && exit 1; fi'`,
```
Comment (alfrunes) [inline cli/write.go:888]: The CI static check is picking on this line length is 3 characters too long, please the string across two lines?
```suggestion
			`; else echo "Mender not found: Please check that Mender is installed" >&2 && `+
			`exit 1; fi'`,
```

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline

**Review by alfrunes — APPROVED** [review]
Excellent! Thank you :pray:

**Comment by danielskinstad** [comment]
Hey, merge when it's ready so I can rebase on top of it :)

**Comment by alfrunes** [comment]
@danielskinstad Done :rocket:

File: cli/.gitignore [hunk 2]
```diff
@@ -2,3 +2,4 @@ artifact_info
 input.mender.txt
 test.txt
 testkey
+artifact.mender
```
Comment (lluiscampos) [inline cli/.gitignore:5]: For future reference,

This and removing the test artifact (thank you for the cleanup!) should have been done on a separate commit.

**Pull request merged**